src/System.Globalization.Extensions/tests/System.Globalization.Extensions.Tests.builds

```diff
@@ -4,19 +4,17 @@
   <ItemGroup>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Unix</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
+      <TestTFMs>netcoreapp1.0</TestTFMs>
     </Project>
     <Project Include="System.Globalization.Extensions.Tests.csproj">
-      <TestTFMs>net463</TestTFMs>
```

Comment:
This should still be OSGroup=Windows_NT and it looks like we also need to include a Unix default build.